src/thunderbots/software/ai/hl/stp/tactic/receiver_tactic.cpp

```diff
@@ -171,8 +139,12 @@ Angle ReceiverTactic::getOneTimeShotDirection(const Ray& shot, const Ball& ball)
     Vector shot_vector = shot.getDirection();
     Angle shot_dir     = shot.getDirection().orientation();
 
-    Point ball_vel       = ball.velocity();
-    Point lateral_vel    = ball_vel.project(shot_vector.norm().perp());
+    Point ball_vel    = ball.velocity();
```

Comment:
Nit: 

Usually, the convention is to make directions 'Vector's

Reply:
Yup, you're right. Good catch. Fixed.